### DISCUSSION TITLE
Design Thinking: Phrasing a HowMightWe question

### DISCUSSION THREAD

**Discussion by Emmy-Emberverse** [GECO-Project-App/facilitation-app#76] [1 - Ideas, Suggestions, & Requests]
One overarching framework we could use for the energy communities is [Design Thinking](https://designsprintkit.withgoogle.com/methodology/overview).

HowMightWe: https://designsprintkit.withgoogle.com/methodology/phase1-understand/hmw-sharing-and-affinity-mapping

1) Explain the exercise and what a HMW question is
Divergence:
2) Everyone takes 5min to write down individually in silence HMW questions 
3) Everyone chooses the HMW question that feels most relevant to them
4) Share the HMW questions, either one by one or put them on a poster and silent gallery (everyone reads in silence).

Then there can be a process of convergence. This could be through dot voting, with or without a round of quick responses (important to keep speaking time short). And then appoint a working group of at least 2 people that will refine and propose the final HMW question.

**Comment by eliasfaltin** [comment]
Would you say this is the exercise most suited to answering the question, "What type of energy project should our community work on?" @Emmy-Emberverse 

Crazy8 and future press releases seem relevant only once you have picked an idea to work on (that is how I interpret it, at least).
HMW seems to be a bit more open to "finding our why/what." Am I getting this correctly, or do we still have an "opening" for a "purpose finding" or "project finding" exercise?

**Comment by Emmy-Emberverse** [comment]
Yeah, HMW is the A-step, creating a shared vision and purpose. It could
start very broad/long-term, e.g. "how might we become more self-sufficient
in 30 years?" and then you could narrow it down into several subquestions,
"how might we put solar panels on the roof of the school?"

On Tue, Sep 24, 2024 at 4:28 PM eliasfaltin ***@***.***>
wrote:

> Would you say this is the exercise most suited to answering the question,
> "What type of energy project should our community work on?"
> @Emmy-Emberverse <https://github.com/Emmy-Emberverse>
>
> Crazy8 and future press releases seem relevant only once you have picked
> an idea to work on (that is how I interpret it, at least).
> HMW seems to be a bit more open to "finding our why/what." Am I getting
> this correctly, or do we still have an "opening" for a "purpose finding" or
> "project finding" exercise?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GECO-Project-App/facilitation-app/discussions/76#discussioncomment-10739980>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLNOJCKAGOP53T35SCJ63FTZYFZJNAVCNFSM6AAAAABOV7WWIGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZTHE4TQMA>
> .
> You are receiving this because you were mentioned.Message ID:
> <GECO-Project-App/facilitation-app/repo-discussions/76/comments/10739980@
> github.com>
>


-- 
*EMMY PATER* | Regenerative Learning Developer & Facilitator
Mobile: +31629295478
Email: ***@***.***
Get in touch on LinkedIn <https://www.linkedin.com/in/emmy-pater-79794776/>
